hash/msx1_cass.xml

```diff
@@ -88,15 +92,98 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<!-- Included with the British HX-10 -->
+	<software name="breakout">
+		<description>Breakout! The Great computer Adventure (United Kingdom)</description>
+		<year>1985</year>
+		<publisher>Toshiba</publisher>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;. The game requires 64k in slot 2"/>
+
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="64019">
+				<rom name="break out (1985)(toshiba-emi)(jp)[needs 64k in slot 2][run'cas-'].cas" size="64019" crc="89517540" sha1="7f5a140d241329f5184635f5567715e0a9f7860b"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Included with the British HX-10 -->
+	<software name="breakouta" cloneof="breakout">
+		<description>Breakout! The Great computer Adventure (United Kingdom, alt)</description>
+		<year>1985</year>
+		<publisher>Toshiba</publisher>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;. The game requires 64k in slot 2"/>
+
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="63987">
+				<rom name="break out (1985)(toshiba-emi)(jp)[a][needs 64k in slot 2][run'cas-'].cas" size="63987" crc="994eb452" sha1="84d22f3c7066f9dea1947ad5fc73745c3d7b95f3"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Included with the British HX-10 -->
+	<software name="breakoutb" cloneof="breakout">
+		<description>Breakout! The Great computer Adventure (United Kingdom, alt 2)</description>
+		<year>1985</year>
+		<publisher>Toshiba</publisher>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;. The game requires 64k in slot 2"/>
+
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="63987">
+				<rom name="break out (1985)(toshiba-emi)(jp)[a2][needs 64k in slot 2][run'cas-'].cas" size="63987" crc="0f2f5296" sha1="e345fa1186c2288dc28d7a02d365461f0cfd67e6"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
“The game requires 64k in slot 2” is potentially confusing for someone unfamiliar with MSX architecture.  I assume this means it requires a system which has the entire 64kB of slot 2 (i.e. all four 16kB pages) occupied by RAM.  However it could be interpreted as meaning it requires a 64kB cartridge of some kind to be inserted in the second cartridge slot.  It would be better to make it unambiguous.

Reply:
Changed the usage info to state that the software was intended to be ran on the hx10 driver and that it will fail on most other systems.